configs/reports/configs/pt-reports.yml

```diff
@@ -1678,3 +1678,58 @@ ReportDefinitions:
     WHERE pi.tenantid != 'uk.testing'
```

Comment:
This should be a single query with the UNION of both, BBPS and non-BBPS sub-queries, not as a separate report for BBPS counts.